.circleci/config.yml

```diff
@@ -84,7 +84,18 @@ jobs:
       - run:
           name: Run Tests
           command: npm run test:ci
-  test-e2e:
+  test-e2e-linux:
```

Comment:
Once I 'touched' this file to remove node version `12` and include `16` for some of the jobs, circle CI started complaining of 
```
Job 'test-e2e-macos' requires 'test-e2e-linux', which is the name of 2 other jobs in workflow 'test-and-publish'
You can give a job within a workflow an explicit name by adding a `name` key
```
hence the change.
[link](https://app.circleci.com/pipelines/github/particle-iot/particle-cli/182)